src/java.desktop/share/legal/lcms.md

```diff
@@ -100,4 +100,4 @@ Dimitrios Anastassakis
 Lemke Software
 Tim Zaman
 Amir Montazery and Open Source Technology Improvement Fund (ostif.org), Google, for fuzzer fundings.
-```
\ No newline at end of file
+```
```

Comment:
This introduces an empty trailing line. I see you have removed trailing whitespace elsewhere.

Was this intentional, to avoid the file ending with the three ticks?